tests/test_layer.py

```diff
@@ -303,5 +305,46 @@ def helper_test_update_mount_points_by_file(layer):
             assert type(e) == SherlockUpdateMountPointsByFileError
 
 
+def helper_test_delete_all_mount_points(layer):
+    """Test delete_all_mount_points API."""
+    try:
+        layer.delete_all_mount_points(
+            "",
+            "Card",
+        )
+        pytest.fail("No exception thrown when using an invalid parameter")
+    except SherlockDeleteAllMountPointsError as e:
+        assert str(e) == "Delete mount points error: Project name is invalid."
+
+    try:
+        layer.delete_all_mount_points(
+            "Test",
```

Comment:
Given that this is a non-named parameter, it's better practice to name the string something to indicate what it is. In contrast, the strings in the examples when the connection is up are defined so that it is obvious what the parameters are for.